xml/osc_building.xml

```diff
@@ -196,5 +195,192 @@ The buildroot was: /var/tmp/build-root/openSUSE_Tumbleweed-x86_64</screen>
+        <keycap>:</keycap>
+        <keycap>q</keycap>
+        <keycap>!</keycap>
+       </keycombo>.
+      </para>
+     </step>
+    </substeps>
+   </step>
+   <step>
+    <para>
+     Create a new <filename>appimage.yml</filename> with the following
+     contents:
+    </para>
+    <example>
+     <title>&obs.ai; from RPM Files (<filename>appimage.yml</filename>)</title>
+     <screen language="yaml">app: <replaceable>APPNAME</replaceable> <co xml:id="co.obs.building.ai.app"/>
+
+build: <co xml:id="co.obs.building.ai.build"/>
+  packages:
+    - linuxdeployqt
+
+ingredients: <co xml:id="co.obs.building.ai.ingredients"/>
+  packages:
+    - <replaceable>MY_RPM_PACKAGE</replaceable>
+
+script: <co xml:id="co.obs.building.ai.script"/>
+  # instructions
+     </screen>
+     <calloutlist>
+      <callout arearefs="co.obs.building.ai.app">
+       <para>
+        <remark>toms 2017-09-05: any restrictions in naming?</remark>
+        Contains the name of &obs.ai; app. You can use any name you want,
+        but it is recommended to use lowercase names.
+       </para>
+      </callout>
+      <callout arearefs="co.obs.building.ai.build">
+       <para>Defines the build requirements as RPM package names.</para>
+      </callout>
+      <callout arearefs="co.obs.building.ai.ingredients">
+       <para>
+        Defines the installation requirements for the &obs.ai;. These
+        packages are installed into the &obs.ai; build root environment.
+       </para>
+      </callout>
+      <callout arearefs="co.obs.building.ai.script">
+       <para>
+        Contains installation instructions.
+        <remark>toms 2017-09-04: FIXME: add more information</remark>
+       </para>
+      </callout>
+     </calloutlist>
+    </example>
+   </step>
+   <step>
+    <para>Build the &obs.ai;:</para>
+    <screen>&prompt.user;<command>osc</command> build AppImage x86_64 appimage.yml</screen>
+   </step>
+   <step>
+    <para>
+     Retrieve the new &obs.ai; from the <filename class="directory"
+      >/var/tmp/build-root/AppImage-x86_64/usr/src/packages/OTHER/</filename>
+     directory under der name <filename><replaceable
+      >APPNAME</replaceable>-<replaceable
+       >VERSION</replaceable>-<replaceable
+        >LIBRARY</replaceable>-<replaceable
+         >ARCH</replaceable>.AppImage</filename>.
+    </para>
+   </step>
+  </procedure>
+  <para>
+   If you want to build &obs.ai;s from source, you need to change the
+   <filename>appimage.yml</filename> file as follows:
+  </para>
+  <example>
+   <title>&obs.ai; from Source File (<filename>appimage.yml</filename>)</title>
+   <screen language="yaml">app: <replaceable>APPNAME</replaceable>
+
+build:
+  packages:
+    - linuxdeployqt
+    - cmake
+  git:
+    - &gitupstream1;.git
+
+script:
+   - cmake -DCMAKE_INSTALL_PREFIX=/usr .
+   - make
+   - make install DESTDIR=/usr
+   # other lines suppressed</screen>
+  </example>
+  <remark>toms 2017-09-05: do we need a _service file?</remark>
```

Comment:
only when using the git upstream integration.

Reply:
Ok, got it, thanks! I've corrected this in the code.